packages/controller/src/main.ts

```diff
@@ -3005,6 +3005,21 @@ async function processMessage(msg: ioBroker.SendableMessage): Promise<null | voi
             restart(() => !isStopping && stop(false));
             break;
         }
+
+        case 'sendToSentry': {
+            const message: string = msg.message.message;
```

Comment:
The message just contains the object id ... is this object-id complete? So also contains adapter and namespace?

Reply:
no just the message without logging namespace, we now send adapter name and version in extra